package.json

```diff
@@ -180,6 +180,7 @@
     "packages": [
       ".",
       "examples",
+      "docs",
```

Comment:
I think that we don't want to include the `docs` in workspaces. Maybe it would be better to treat the project as "submodule" (go into `docs/` and then do `npm i` and `npm run` stuff). WDYT?

cc @wojciechczerniak

Reply:
We can do that, but it would be harder for the `test:ci` script to trigger the docs' tests (if there will be any).

Reply:
Currenty, the docs does not have any tests but if the have any they could by managed by GH Workflow. As the „Documentation lint” process is done (https://github.com/handsontable/handsontable/blob/feature/issue-7624/.github/workflows/documentation-linter.yml). The workflow is triggered only for changes pushed to the „docs/*”

![2FAA6977-7B30-4272-ADE6-47327D117B35](https://user-images.githubusercontent.com/571316/122251095-7feebc80-ceca-11eb-8bcd-93084253c747.jpeg)

Reply:
Docs in workspace have value if they have Handsontable in dependencies and we want to run them with latest build. But AFAIK on a localhost they just use relative path to `../dist`, not the package

Reply:
Makes sense, I removed it from the workspaces and added it to the ignored path list in the `test:ci` script: b09721a.

I kept the rest of the changes because they should be useful to the `examples` project.